cellprofiler/modules/unmixcolors.py

```diff
@@ -31,7 +31,7 @@
 ^^^^^^^^^^^^^^^
 
 This code is adapted from the ImageJ plugin,
-*Colour\_Deconvolution.java* (described `here`_) written by A.C.
+*Colour_Deconvolution.java* (described `here`_) written by A.C.
```

Comment:
Link referenced here (link text at L46) is dead, please replace with http://imagej.net/Colour_Deconvolution